doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1541,13 +1541,19 @@ whose general syntax is
    :name: =>
 
 .. prodn::
-   i_item ::= @i_pattern %| @s_item %| @clear_switch %| {? %{%} } /@term
+   i_item ::= @i_pattern %| @s_item %| @clear_switch %| @i_view %| @i_block
```

Comment:
You need to update the paragraph saying "The last entry in the i_item grammar rule, /term, represents a view"

Reply:
done